rules/common.smk

```diff
@@ -39,19 +51,25 @@ else:
             sequences = "data/{segment}/sequences.fasta",
         params:
             s3_src=S3_SRC,
+            no_sign_request=no_sign_request,
         shell:
             """
-            aws s3 cp {params.s3_src:q}/{wildcards.segment}/sequences.fasta.zst - | zstd -d > {output.sequences}
+            aws s3 cp {params.no_sign_request} \
```

Comment:
```suggestion
            aws s3 cp {params.no_sign_request:q} \
```

Keep consistent with our [style guide](https://docs.nextstrain.org/en/latest/reference/snakemake-style-guide.html#always-use-quoted-q-interpolations)?

Reply:
Included in https://github.com/nextstrain/avian-flu/pull/49/commits/d1ca49c6c26012a56060e7f9e43285ea2e86e0f2